plume-file-web-upload-jersey/README.md

```diff
@@ -14,21 +14,44 @@ Setup
   <artifactId>plume-file-web-upload-jersey</artifactId>
 </dependency>
 ```
+2. Create the webservice file with the injected dependency `FileMimeTypeDetector` (it comes from Plume file Core)
+
+Uploading with Multipart
+------------------------
+
+Using the Jersey Multipart API is the easiest way to upload a file.
+This API allows you to get the file InputStream directly from the webservice parameters, by reading the multipart content.
+
+This method preferably used for small files (< 100Mo).
```

Comment:
Il faudrait ajouter : This method however has a performance drawback: it will first copy the uploaded file to a temporary file on disk, before giving access to this temporary file.